lib/vet360/service.rb

```diff
@@ -46,7 +46,7 @@ def log_error_messages(error)
         log_message_to_sentry(
           error.message,
           :error,
-          { url: config.base_path, body: error.body },
+          extra_context: { url: config.base_path, body: error.body },
```

Comment:
You actually won't need to pass the `extra_context ` hash.  For example, this call (without the `extra_context`) has three keys in a hash that are presented directly as header titles in Sentry's "Additional Data" section:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/concerns/vet360/writeable.rb#L31-L35

This directly presents those keys in Sentry.  Note the headers for **controller**, **response** and **params**:

![image](https://user-images.githubusercontent.com/7482329/44037378-6b2eff48-9ed1-11e8-9bc0-99cba2717edf.png)

So it won't hurt, nor does it necessarily help.  I'll tell you that sometimes people use it, sometimes people don't.  Personally, I lean towards leaving the extra layer of nesting out.  In particular if/when Sentry allows for searching/filtering by those headers, the extra layer of nesting will make that a bit more problematic.

Reply:
This makes sense to me. I opened this PR as a result of a slack conversation with @dhall this morning but I can see why adding another layer of nesting with the key could impede grepping for results in Sentry, but my only concern would be the inconsistency, i.e. on [line 30](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/vet360/service.rb#L30) or [line 53](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/vet360/service.rb#L53) where the extant code **is** utilizing the nested block. Perhaps we need to define a common pattern when it makes sense to use this block or when it's better to leave it out?

Reply:
Only stood out to me due to the inconsistency within the single method. If it the additional layer isn't helpful I don't have a problem dropping it but I think it should be handled consistently in this PR :)